app/models/user.rb

```diff
@@ -676,7 +676,7 @@ def publish_user_status(status)
       payload = nil
     end
 
-    MessageBus.publish("/user-status/#{id}", payload, user_ids: [id])
+    MessageBus.publish("/user-status", { id => payload })
```

Comment:
This will make the data public, even to anon. Is that what we want? 

If not, we should publish to `::Group::AUTO_GROUPS[:trust_level_0]`

Reply:
My understanding is that we're on the fence yet, we may decide to show status messages to anon users. But anyway, publishing status to trust_level_0 users is better start and better default behavior.